pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -90,7 +90,7 @@ var (
 	// CheckServantJobPeriod defines the time interval between two successive ServantJob statu's inspection
 	CheckServantJobPeriod = time.Second * 10
 	// ValidServerVersions contains all compatible server version
-	// yurtctl only support Kubernetes 1.12+ - 1.16+ for now
+	// yurtctl only support Kubernetes 1.12+ - 1.23+ for now
 	ValidServerVersions = []string{
```

Comment:
This variable seems to have the same effect as the `ValidServerVersions` in `openyurt/pkg/yurtctl/cmd/convert/options.go`. How about merging the two variables together？

Reply:
fixed.